pkg/server/container_start.go

```diff
@@ -288,6 +290,27 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 	return g.Spec(), nil
 }
 
+// setOCIProcessArgs sets process args. It returns error if the final arg list
+// is empty.
+func setOCIProcessArgs(g *generate.Generator, config *runtime.ContainerConfig, imageConfig *imagespec.ImageConfig) error {
+	command, args := config.GetCommand(), config.GetArgs()
+	// The following logic is migrated from https://github.com/moby/moby/blob/master/daemon/commit.go
+	// TODO(random-liu): Figure out whether the following logic is
+	if len(command) == 0 {
```

Comment:
This is the current behavior of docker, I don't fully understood it, either. However, there is no definition of how this should behave, so I just reuse the docker logic.

The TODO is to clearly define this and fix the logic here.